frontend/src/index.tsx

```diff
@@ -1,17 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import FooTouer from './components/FooTouer/FooTouer';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <FooTouer />
```

Comment:
@nowakprojects , to jest dobre zagranie? To lepsze niż "App"?

Reply:
Chyba wszystko jedno, App jest defaultowo.

Reply:
Jak ta nasza apka się w końcu nazywa. Była zmiana :P ?

Reply:
Wydaje mi się, że tak było na designach w figmie. W takim razie proszę o potwierdzenie/informację jak nazywa się nasza apka 👯

Reply:
Nazwa błędna